### ISSUE TITLE
Game Center Only ever sync's with the cloud once.

### ISSUE THREAD

**Issue by danoli3** [nihalahmed/GameCenterManager#56]
Doing some tests and in testing the OSX Game Center never successfully syncs.

I'll find the cause.

Initial Findings:
```
[AVConference]: GKSConnSettings: set server: {
	    "gk-cdx" = "17.155.127.221:4398";
	    "gk-commnat-cohort" = "17.155.127.223:16386";
	    "gk-commnat-main0" = "17.155.127.222:16384";
	    "gk-commnat-main1" = "17.155.127.222:16385";
	}
14:34:07.092362 [NSLog]: The requested operation could not be completed due to an error communicating with the server.
```

**Comment by flexgames** [comment]
Fantastic. Can't wait to see what you find out.

**Comment by danoli3** [comment]
Okay I think I found it

-  [NSLog]: The requested operation could not be completed due to an error communicating with the server.
- <Warning>: 14:54:02.346662 [NSLog]: GC Error: Error Domain=GKErrorDomain Code=3 
  - "The requested operation could not be completed due to an error communicating with the server." UserInfo={GKServerStatusCode=5044, NSLocalizedDescription=The requested operation could not be completed due to an error communicating with the server., NSUnderlyingError=0x60000004b550 {Error Domain=GKServerErrorDomain Code=5044 "status = 5044, **AppName does not support achievements**" UserInfo={GKServerStatusCode=5044, NSLocalizedFailureReason=status = 5044, AppName does not support achievements}}}

**Comment by danoli3** [comment]
Which was likely due to this Project issue:

<img width="703" alt="screen shot 2015-11-13 at 2 59 50 pm" src="https://cloud.githubusercontent.com/assets/830748/11138507/4335f946-8a17-11e5-84d2-84a8c18aa431.png">

**Comment by danoli3** [comment]
I think this one was isolated to my project. Closing.

**Comment by danoli3** [comment]
Ah... facepalm

<img width="984" alt="screen shot 2015-11-13 at 3 03 40 pm" src="https://cloud.githubusercontent.com/assets/830748/11138547/c45e55ea-8a17-11e5-9679-66e860fce6da.png">

**Comment by danoli3** [comment]
Actually despite this, it still only ever syncs once.

**Comment by danoli3** [comment]
Fixed with #57
